cardano-crypto-class/src/Cardano/Crypto/DSIGN/SchnorrSecp256k1.hs

```diff
@@ -186,14 +186,17 @@ instance DSIGNAlgorithm SchnorrSecp256k1DSIGN where
   rawSerialiseSignKeyDSIGN (SignKeySchnorrSecp256k1 skPSB) = psbToByteString skPSB
   {-# NOINLINE rawDeserialiseVerKeyDSIGN #-}
   rawDeserialiseVerKeyDSIGN bs = 
-    unsafeDupablePerformIO . unsafeUseAsCStringLen bs $ \(ptr, _) -> do
-      let dataPtr = castPtr ptr
-      (vkPsb, res) <- psbCreateSizedResult $ \outPtr ->
-        withForeignPtr secpCtxPtr $ \ctx -> 
-          secpXOnlyPubkeyParse ctx outPtr dataPtr
-      pure $ case res of 
-        1 -> pure . VerKeySchnorrSecp256k1 $ vkPsb
-        _ -> Nothing
+    unsafeDupablePerformIO . unsafeUseAsCStringLen bs $ \(ptr, len) ->
+      if len /= 32
```

Comment:
I think there is a way to get `32` out of `SchnorrSecp256k1DSIGN`, I think it's `SizeVerKeyDSIGN`.

Reply:
That involves `natVal`ing a `Proxy`. I'm happy to make this change though.

Reply:
if it's not a huge inconvenience,  that would be grand!